rust-toolchain.toml

```diff
@@ -1,2 +1,6 @@
 [toolchain]
 channel = "{{ toolchain }}"
+{%- if toolchain == "nightly" %}
+components = ["rust-src"]
+targets = ["riscv32imc-unknown-none-elf"]
```

Comment:
maybe we should also have a variable for this since for ESP32-C6 and later we probably want `riscv32imac-unknown-none-elf`

Reply:
True, in fact, we already have it https://github.com/esp-rs/esp-template/blob/main/pre-script.rhai#L48